app/components/layer-image-text/style.scss

```diff
@@ -1,6 +1,6 @@
-.image img {
-    width: 100%;
-    overflow: hidden;
+img {
+    width:500px;
```

Comment:
I see why you did this, but this makes the image not responsive, especially on small windows. Can you test and check for alternatives.